Modules/IO/GDCM/test/Input/Lily/HTJ2K-YBR_ICT.dcm.cid

```diff
@@ -0,0 +1 @@
+bafkreib5e6kni2tdbtaxpz2i77k7hp5gzkztppiuws3ov4yx4dswlna6p4
```

Comment:
How large are they?

Reply:
Some examples files are available here https://github.com/chafey/DICOMHTJ2K. But caution, they are not correct (s. issue https://github.com/chafey/DICOMHTJ2K/issues/4). S. pull request https://github.com/chafey/DICOMHTJ2K/pull/5 for correct files. Here are also fixed files https://github.com/hyldliu/DICOMHTJ2K.
These "Lily" files might be mine, not sure, should be OK too. BTW, it is most likely willowherb :)